homeassistant/components/light/hue.py

```diff
@@ -23,6 +23,7 @@
 
 DEPENDENCIES = ['hue']
 SCAN_INTERVAL = timedelta(seconds=5)
+PARALLEL_UPDATES = 0
```

Comment:
This is already the default since there's an `async_setup_platform` name in the module.

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity_platform.py#L58

Am I missing something?

Reply:
Oh snap you're right. I thought I finally figured out why it was messing up.